internal/witcli/dir_test.go

```diff
@@ -0,0 +1,71 @@
+package witcli
```

Comment:
would be nice to add a test case where it tries to create a diretory where permissions are restricted. 

E.g.

```Go

func TestFindOrCreateDirPermissionDenied(t *testing.T) {
	if runtime.GOOS == "windows" {
	    return
	}
	_, err := FindOrCreateDir("/root/protected")
	if err == nil || !os.IsPermission(err) {
		t.Errorf("Expected permission error, got: %v", err)
	}
}
```

Reply:
I like that suggestion, added one 👍